### DISCUSSION TITLE
关于NeuralNetworkDetect中threshold的问题

### DISCUSSION THREAD

**Discussion by aharry-1** [MaaXYZ/MaaFramework#365] [Q&A]
我把NeuralNetworkDetect中的threshold设为了0.9，但是仍然会在score=0.3的时候成功执行点击

![屏幕截图 2024-09-28 042433](https://github.com/user-attachments/assets/1a12f54d-74fa-49fb-a3c2-d6cf85a5303e)
![image](https://github.com/user-attachments/assets/7b74dbf0-d6f4-4743-a16a-e19358c8a179)

请问如果只想score>=0.9时候执行点击，score<0.9时触发失败该怎么设置？

**Comment by MistEO** [comment]
log 发一下，debug/maa.log

**Reply by MistEO** [reply]
不用发了，是 bug，下个版本改一下 https://github.com/MaaXYZ/MaaFramework/issues/367
